services/notify/src/api.js

```diff
@@ -215,3 +216,26 @@ builder.declare({
     continuationToken: query.continuation || undefined,
   });
 });
+
+builder.declare({
```

Comment:
If this is going into production don't we want to scope protect this? I know it is just a throwaway table but it seems nice to not let arbitrary access. We could also `noPublish` this maybe?

Reply:
Good point!  It turns out noPublish means that the method isn't even in the generated API clients, which wasn't very useful.  I don't know why that exists at all tbh.